src/ui/flutter_app/android/app/build.gradle

```diff
@@ -64,22 +49,19 @@ android {
         }
     }
 
-    signingConfigs {
-        release {
-            keyAlias keystoreProperties['keyAlias']
-            keyPassword keystoreProperties['keyPassword']
-            storeFile keystoreProperties['storeFile'] ? file(keystoreProperties['storeFile']) : null
-            storePassword keystoreProperties['storePassword']
-        }
-    }
+
     buildTypes {
         debug {
             signingConfig signingConfigs.debug
-            //applicationIdSuffix ".debug"
-            //debuggable true
+            minifyEnabled true
```

Comment:
Does adding shrinking cause the symbols to be removed from the Release version of the apk, making it difficult to backtrack and analyze? In fact, this is open-source software and there is no need to obfuscate it if not to reduce the size of the APK. I wonder how much turning on this switch would reduce the size of the APK though? Need to test it.